miso-web/src/main/webapp/pages/editRun.jsp

```diff
@@ -696,7 +696,7 @@
     <c:choose>
     <c:when test="${not empty run.platformType}">
       Run.pool.poolSearch("", '${run.platformType.key}');
-      <c:if test="${run.id != 0}">
+      <c:if test="${run.id != 0 and metrixEnabled}">
```

Comment:
Yes, but only a noticeable problem when a run is missing a status, which normally doesn't happen. I noticed it when generating runs before I was also generating status for them. There's another check for metrixEnabled for querying the interop files. This one is for updating run status